Noonbody/Noonbody/Source/Presentation/Panorama/ViewControllers/PanoramaViewController.swift

```diff
@@ -64,12 +64,12 @@ class PanoramaViewController: BaseViewController {
     let cellSpacing: CGFloat = 2
     private let viewModel = PanoramaViewModel(panoramaUseCase: DefaultPanoramaUseCase(panoramaRepository: DefaultPanoramaRepository()))
     private var popupViewController = PopUpViewController(type: .delete)
-    private var bodyPart = 0
     private var albumId: Int
     private var albumData: Album
-    var deleteData: [Int] = [] {
+    var bodyPart = 0
+    var deleteData: [Int: Int] = [:] {
         didSet {
-            navigationItem.rightBarButtonItem?.isEnabled = !deleteData.isEmpty ? true : false
+            navigationItem.rightBarButtonItem?.isEnabled = !deleteData.isEmpty || !editMode ? true : false
```

Comment:
isEmpty, eidtMode 자체가 Bool 값이기 때문에 다음과 같이 써줄 수 있을 것 같습니다
```suggestion
            navigationItem.rightBarButtonItem?.isEnabled = !deleteData.isEmpty || !editMode
```